sources/packages/web/src/composables/useAssessment.ts

```diff
@@ -39,5 +40,20 @@ export function useAssessment() {
     }
   };
 
-  return { mapRequestAssessmentChipStatus, mapAssessmentHistoryChipStatus };
+  const mapAssessmentDetailHeader = (
+    assessment: AssessmentDetailHeader,
+  ): Record<string, string> => {
+    return {
+      "Application #": assessment.applicationNumber,
+      Institution: assessment.institutionName,
+      "Study dates": `${assessment.offeringStudyStartDate} - ${assessment.offeringStudyEndDate}`,
```

Comment:
I noticed that this date is coming formatted from the API. I would still prefer that the API sends the date-only format and methods like this would be adapting it to the UI format.

Reply:
It was when we were sending the values to form.io. Still not a strong reason IMO but that was the argument provided, I guess.